benchmarks/Fractions.Benchmarks/ComparisonBenchmarks.cs

```diff
@@ -54,26 +54,30 @@ public class ComparisonBenchmarks {
         yield return [
             Fraction.FromDoubleRounded(Math.PI),
             Fraction.NegativeInfinity];
+        // very close fractions: 
+        yield return [
+            new Fraction(12.3456789987654321m),
+            new Fraction(12.3456789987654322m)];
```

Comment:
Had to add another "extreme" value, as to not make it seem like these operations are `completely-allocation-free`. 😄